Maple2.Server.Game/Manager/ItemEnchantManager.cs

```diff
@@ -7,18 +7,18 @@
 
 namespace Maple2.Server.Game.Manager;
 
-public class ItemEnchantManager {
+public class ItemEnchantManager(GameSession session, Lua.Lua lua) {
     private const int MAX_RATE = 100;
     private const int MAX_EXP = 10000;
     private const int CHARGE_RATE = 1;
 
     // ReSharper disable RedundantExplicitArraySize
-    private static readonly int[] RequireFodder = new int[15] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 2, 3, 3, 4 };
```

Comment:
The 15 here is explicit because this MUST be size 15 or there is probably some bug. We should keep it that way intentionally.